backend/tests/Feature/Webhooks/WebhookDeliveryTest.php

```
@@ -0,0 +1,177 @@
+        Webhook::factory()->subscribedTo([WebhookEventRegistry::ISSUE_CREATED])->create();
+
+        app(WebhookDispatcher::class)->dispatch(WebhookEventRegistry::BATCH_COMPLETED, ['id' => 1]);
+
+        $this->assertDatabaseCount('webhook_deliveries', 0);
+        Bus::assertNotDispatched(DeliverWebhookJob::class);
+    }
+
+    public function test_work_order_status_change_fires_webhook(): void
+    {
+        Bus::fake();
+        Webhook::factory()->subscribedTo([WebhookEventRegistry::WORK_ORDER_STATUS_CHANGED])->create();
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\Webhook::factory().


(staticMethod.notFound)